app/views/admin/promotional_feature_items/confirm_destroy.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :context, @organisation.name %>
+<% content_for :page_title, "Delete #{@promotional_feature_item.title}" %>
+<% content_for :title, "Delete #{@promotional_feature_item.title}" %>
+<% content_for :title_margin_bottom, 6 %>
+
+<% page_title "Feature item for #{@promotional_feature_item.title}" %>
```

Comment:
We should only ever be hitting this page when we use the GOV.UK Design System so we don't need to worry about setting the page title for the bootstrap implementation here